node/cn/handler.go

```diff
@@ -1083,80 +1080,79 @@ func (pm *ProtocolManager) broadcastCNTx(txs types.Transactions) {
 		half := (len(peers) / 2) + 2
 		peers = samplingPeers(peers, half)
 		for _, peer := range peers {
-			txset[peer] = append(txset[peer], tx)
+			cnPeersWithoutTxs[peer] = append(cnPeersWithoutTxs[peer], tx)
 		}
 		logger.Trace("Broadcast transaction", "hash", tx.Hash(), "recipients", len(peers))
 	}
 
-	propTxPeersGauge.Update(int64(len(txset)))
+	propTxPeersGauge.Update(int64(len(cnPeersWithoutTxs)))
 	// FIXME include this again: peers = peers[:int(math.Sqrt(float64(len(peers))))]
-	for peer, txs := range txset {
+	for peer, txs2 := range cnPeersWithoutTxs {
 		//peer.SendTransactions(txs)
-		peer.AsyncSendTransactions(txs)
+		peer.AsyncSendTransactions(txs2)
 	}
 }
 
-func (pm *ProtocolManager) broadcastNoCNTx(txs types.Transactions, resend bool) {
-	var cntxset = make(map[Peer]types.Transactions)
-	var txset = make(map[Peer]types.Transactions)
+func (pm *ProtocolManager) broadcastTxsFromPN(txs types.Transactions) {
+	cnPeersWithoutTxs := make(map[Peer]types.Transactions)
+	peersWithoutTxs := make(map[Peer]types.Transactions)
 	for _, tx := range txs {
 		// TODO-Klaytn drop or missing tx
-		if resend {
```

Comment:
Is it safe not to check `resend` case?

Reply:
@jeonghonah `resend == true` case was only called from `ReBroadcastTxs` and now `ReBroadcastTxs` has its own function body. As you can see, `resend` flag completely separates codes inside `broadcastNoCNTx` by the if statements. Therefore, there's no reason to use the flag inside the function.

Reply:
@jeonghonah @KimKyungup Please check the before/after function logics since this change is a kind of  large and complicated change.